models/ResumeModel.ts

```diff
@@ -13,7 +13,7 @@ export class ResumeModel extends BaseEntity {
   user: UserModel;
 
   @Column('boolean', { default: true, nullable: false })
-  isResumeVisible: boolean;
+  isResumeVisible: boolean = false;
 
```

Comment:
Also, this is not a change in your PR, but in `ResumeModel::getPublicResume()` we need to return the entire user, and not just their uuid. You can see in the figma below that we need to return some user data to the frontend (in this case, just first and last names):
![image](https://user-images.githubusercontent.com/33337209/198407460-e7f8a2a9-35dc-4187-8dcb-ae5c492e497c.png)

Can we add this change into this PR? Instead of returning just `user: this.user.uuid`, let's return the user as a `PublicUser`. This means we also need to do a join when querying all visible resumes. Let's also test that we're returning the proper user object in our tests as well (just making sure the object is returned instead of just a uuid)